includes/class-wc-logger.php

```diff
@@ -139,11 +139,14 @@ public function log( $level, $message, $context = array() ) {
 		}
 
 		if ( $this->should_handle( $level ) ) {
-			$timestamp = current_time( 'timestamp', 1 );
-			$message   = apply_filters( 'woocommerce_logger_log_message', $message, $level, $context );
+			$timestamp = time();
 
 			foreach ( $this->handlers as $handler ) {
-				$handler->handle( $timestamp, $level, $message, $context );
+				$message = apply_filters( 'woocommerce_logger_log_message', $message, $level, $context, $handler );
```

Comment:
This filter requires a docblock now since there's a new parameter, it need to be proper documented.

Reply:
Added. Chose 5.2 since it's next and would be good to get it in if possible. Otherwise I can update with a point release or something.

Reply:
Thanks for adding the docblock. We're past the code freeze on 5.2 by quite a bit. The freeze for 5.3 is this Thursday, the 15th. Can this ship with 5.3 when it's released on the 11th of May?

Reply:
Updated